shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -573,7 +578,13 @@ - (SemanticsObject*)search:(CGPoint)point {
 // IOS 16. Overrides this method to focus the first eligiable semantics
 // object in hit test order.
 - (id)_accessibilityHitTest:(CGPoint)point withEvent:(UIEvent*)event {
-  return [self search:point];
+  id hittest = [self search:point];
+  if (hittest != self) {
```

Comment:
can we just check if hittest is a semanticObject here?

Reply:
sure. done